src-java/testing/performance-tests/src/test/groovy/org/openkilda/performancetests/spec/EnduranceV2Spec.groovy

```diff
@@ -175,18 +178,127 @@ idle, mass manual reroute, isl break. Step repeats pre-defined number of times"
         //define payload generating method that will be called each time flow creation is issued
         makeFlowPayload = {
             def flow = flowHelperV2.randomFlow(*topoHelper.getRandomSwitchPair(), false, flows)
-            flow.maximumBandwidth = 100000
+            flow.maximumBandwidth = 200000
             return flow
         }
         //'dice' below defines events and their chances to appear
         dice = new Dice([
-                new Face(name: "delete flow", chance: 10, event: { deleteFlow() }),
-                new Face(name: "update flow", chance: 20, event: { updateFlow() }),
-                new Face(name: "create flow", chance: 10, event: { createFlow(makeFlowPayload(), true) }),
-                new Face(name: "blink isl", chance: 25, event: { blinkIsl() }),
+                new Face(name: "delete flow", chance: 19, event: { deleteFlow() }),
+                new Face(name: "update flow", chance: 0, event: { updateFlow() }),
+                new Face(name: "create flow", chance: 19, event: { createFlow(makeFlowPayload(), true) }),
+                new Face(name: "blink isl", chance: 32, event: { blinkIsl() }),
                 new Face(name: "idle", chance: 0, event: { TimeUnit.SECONDS.sleep(3) }),
-                new Face(name: "manual reroute 5% of flows", chance: 10, event: { massReroute() }),
-                new Face(name: "break isl", chance: 25, event: { breakIsl() })
+                new Face(name: "manual reroute 5% of flows", chance: 0, event: { massReroute() }),
+                new Face(name: "break isl", chance: 30, event: { breakIsl() })
+        ])
+        debugText = preset.debug ? " (debug mode)" : ""
+    }
+
+    @Unroll
+    def "Random events appear on existing env for certain period of time"() {
+        Assume.assumeThat(preset.debug, equalTo(debug))
+
+        given: "A live env with certain topology deployed and existing flows"
+        setTopologies(topoHelper.readCurrentTopology())
+        flows.addAll(northbound.getAllFlows().collect { flowHelperV2.toV2(it) })
+
+        when: "With certain probability one of the following events occurs: flow creation, flow deletion, isl blink, \
+idle, mass manual reroute, isl break. Step repeats for pre-defined amount of time"
+        Wrappers.timedLoop(preset.durationMinutes * 60) {
+            log.debug("running event #$it")
+            dice.roll()
+            TimeUnit.SECONDS.sleep(preset.pauseBetweenEvents)
+        }
+
+        and: "Wait for any blinking isls to get UP and flows to finish rerouting"
+        Wrappers.wait(antiflap.getAntiflapCooldown() + discoveryInterval + WAIT_OFFSET * 3) {
+            def isls = northbound.getAllLinks()
+            (topology.isls - brokenIsls).each {
+                assert islUtils.getIslInfo(isls, it).get().state == IslChangeType.DISCOVERED
+            }
+        }
+        Wrappers.wait(WAIT_OFFSET + flows.size() * 0.8) {
```

Comment:
Could you explain or leave a comment about magic numbers(`60`, `0.8`, `WAIT_OFFSET * 3`) ?
How did you find them?

Reply:
Added a comment. Tbh it is just a big wait which somehow depends on flows amount. Nobody knows the real formula)

Reply:
'60' is for converting minutes to seconds. do we really need a comment there?

Reply:
I just wondered